nerfstudio/process_data/colmap_utils.py

```diff
@@ -455,6 +455,11 @@ def colmap_to_json(
     out = parse_colmap_camera_params(cam_id_to_camera[1])
     out["frames"] = frames
 
+    applied_transform = np.eye(4)[:3, :]
+    applied_transform = applied_transform[np.array([1, 0, 2]), :]
```

Comment:
Just confirming, this is the only change from the last PR?

Reply:
I also changed the tests because the interface of NerfstudioDataParserConfig changed. But otherwise, yes.